src/Components/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,7 +1,72 @@
-import React from 'react'
+import { Typography, TextField, Button, Grid } from '@mui/material';
+import styles from '../SignUp/SignUp.module.scss';
+import signUpImage from "../../assets/Rectangle 8.png"
+import { useForm } from 'react-hook-form';
+import { Link, useNavigate } from 'react-router-dom';
+import axios from 'axios';
 
 export default function ForgetPassword() {
+
+  const navigate = useNavigate()
+  const {
+    register,
+    formState: { errors },
+    handleSubmit
+  } = useForm( { mode: "all" })
+
+  const onSubmit = (data) => {
+    console.log(data);
+    axios.post("http://154.41.228.234:3000/api/v0/portal/users/forgot-password", data
+    ).then(response => {
+      console.log(response.data.message);
+    })
+      .catch(error => {
+        console.error(error.response.data.message);
+      });
+      navigate("/reset-pass")
```

Comment:
This should only happen if the request is success (in the `then` block), you'll notice in the application that if you entered incorrect data that the user will be directed to the reset-pass directly.

Reply:
you mean that the user will be directed to the reset-pass directly as a result of my code now so i should put navigate("/reset-pass") in (then) block to navigate just with correct data?